core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -2383,6 +2384,7 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
 
   private def buildTransactionalProducer(): KafkaProducer[Array[Byte], Array[Byte]] = {
     producerConfig.setProperty(ProducerConfig.TRANSACTIONAL_ID_CONFIG, transactionalId)
+    producerConfig.setProperty(ProducerConfig.ENABLE_IDEMPOTENCE_CONFIG, "true")
```

Comment:
Do we need to explicitly set it?

Reply:
Yes, because at the beginning of the `AuthorizerIntegrationTest` test suite, the idempotence is disabled.

Reply:
You're right, I missed that!